### PR TITLE
Drop all references to Canvas Hit Regions; archive

### PR DESCRIPTION
**Pull request by sideshowbarker**
See also https://github.com/mdn/archived-content/pull/2

### CODE REVIEW

**Comment by Elchi3** [comment]
Nice! Thanks for doing this. First time I see an archiving PR but I guess it makes sense to not outright delete it. 

Questions:
- How do we feel about archiving the reference pages? (I think according to BCD they met irrelevance criteria).
- Are there any other techniques we should people point to for making canvas more accessible?

**Comment by sideshowbarker** [comment]
> guess it makes sense to not outright delete it.

In general I wonder whether we should ever just be outright deleting anything — due to that causing links to break. I think we should be required by default to set up a redirect — except for cases where decide that’s not necessary.

I know deletion is covered in the docs at https://github.com/mdn/content/blob/main/README.md#deleting-a-document but in general should we actually do that without at least also setting up a redirect?

> How do we feel about archiving the reference pages? (I think according to BCD they met irrelevance criteria).

I feel good about archiving them :) I think the more clearly we can make things to developers about what’s supported and recommended, the better. The archiving mechanism seems to help convey that status clearly.

> Are there any other techniques we should people point to for making canvas more accessible?

That’s a good question, though I guess it’s outside the scope of this PR. But that makes me realize we should at least be giving a heads-up about this PR to @mdn/yari-content-accessibility.

**Comment by ericwbailey** [comment]
>I think the more clearly we can make things to developers about what’s supported and recommended, the better. The archiving mechanism seems to help convey that status clearly.

For what its worth, I would like to see overpoweringly obvious messaging about archived/deprecated pages. I completely agree that they should be preserved for archival purposes instead of deletion, but I worry that some people may stumble directly into these pages via a search and not realize it is being preserved for historical context.

A lot of the accessibility-specific content is stale, which runs aground of this issue.

File: files/en-us/web/html/element/canvas/index.html [hunk 1]
```diff
@@ -64,7 +64,7 @@ <h2 id="Usage_notes">Usage notes</h2>
 
 <h3 id="Alternative_content">Alternative content</h3>
 
-<p>You may (and should) provide alternate content inside the <code>&lt;canvas&gt;</code> block. That content will be rendered both on older browsers that don't support canvas and in browsers with JavaScript disabled. Providing a useful fallback text or sub DOM helps to <a href="/en-US/docs/Web/API/Canvas_API/Tutorial/Hit_regions_and_accessibility">make the canvas more accessible</a>.</p>
+<p>You may (and should) provide alternate content inside the <code>&lt;canvas&gt;</code> block. That content will be rendered both on older browsers that don't support canvas and in browsers with JavaScript disabled.</p>
```
Comment (ericwbailey) [inline files/en-us/web/html/element/canvas/index.html:67]:
```suggestion
<p>You should provide alternate content inside the <code>&lt;canvas&gt;</code> block. That content will be rendered both on older browsers that don't support canvas and in browsers with JavaScript disabled.</p>
```

**Review by ericwbailey — CHANGES_REQUESTED** [review]

**Comment by peterbe** [comment]
> > I think the more clearly we can make things to developers about what’s supported and recommended, the better. The archiving mechanism seems to help convey that status clearly.
> 
> For what its worth, I would like to see overpoweringly obvious messaging about archived/deprecated pages. I completely agree that they should be preserved for archival purposes instead of deletion, but I worry that some people may stumble directly into these pages via a search and not realize it is being preserved for historical context.
> 
> A lot of the accessibility-specific content is stale, which runs aground of this issue.

Do you think the yellow banner that appears on archived pages is good enough? E.g. https://developer.mozilla.org/en-US/docs/Mozilla/Thunderbird

**Comment by ericwbailey** [comment]
>Do you think the yellow banner that appears on archived pages is good enough? E.g. developer.mozilla.org/en-US/docs/Mozilla/Thunderbird

Bit of a nitpick, but it might be good to be explicit and state something along the lines of "some of this content may be out of date."

**Comment by peterbe** [comment]
> > Do you think the yellow banner that appears on archived pages is good enough? E.g. developer.mozilla.org/en-US/docs/Mozilla/Thunderbird
> 
> Bit of a nitpick, but it might be good to be explicit and state something along the lines of "some of this content may be out of date."

Good idea! Worth keeping in mind. 
Like on this blog, https://www.peterbe.com/plog/blogitem-20030707-1614  it says 
```
This blog post is 17 years old! Most likely, its content is outdated.
```

**Comment by ericwbailey** [comment]
Yeah, I really like that pattern too!

**Comment by sideshowbarker** [comment]
Dropping this one from plate. If somebody else wants to pick it up, that’s of course fine

**Pull request closed without merging**